components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemBuy.vue

```diff
@@ -8,7 +8,8 @@
             class="w-full"
             content-class="buy-tooltip"
             :position="isMobileDevice ? 'top' : 'left'"
-            :auto-close="!isMobileDevice ? ['outside', 'escape'] : []"
+            :auto-close="['outside', 'escape']"
+            :triggers="[isMobileDevice ? 'click' : 'hover']"
```

Comment:
this seems like generally good logic
can it be placed as default behavior in `NeoTooltip`?

Reply:
@daiagi  yes did it. a little question that currently we use this logic to judge if it's mobile device
```
const isMobileDevice = computed(() => useWindowSize().width.value < 1024)
```
can i import `useWindowSize` from `@vueuse/core` directly in `NeoTooltip.vue`?

Reply:
done